lib/model/query/entities.js

```diff
@@ -96,9 +96,13 @@ const processSubmissionEvent = (event) => (container) =>
     .then((entity) => {
       if (entity != null) {
         return container.Audits.log({ id: event.actorId }, 'entity.create', { acteeId: entity.aux.dataset.acteeId },
-          { entity: { uuid: entity.uuid, label: entity.def.label, dataset: entity.aux.dataset.name },
+          {
+            entityId: entity.id,
+            eventId: event.id,
```

Comment:
I think we're planning not to backfill this. If that's right, could you leave a comment above this property that notes that it wasn't saved for entities created before v2023.3? I think it'll be nice to have some record of how the details schema has changed over time.